src/js/nav/globalNav.js

```diff
@@ -32,7 +32,8 @@ export const grouppedNav = {
                     {
                         id: 'sources',
                         title: 'Sources',
-                        reload: 'settings/sources'
+                        reload: 'settings/sources',
+                        disabled: window.location.pathname.indexOf('/beta') === -1
```

Comment:
You can just use `window.insights.chrome.isBeta()`

Reply:
Word!

Reply:
Oh wait no :(
Try it out. This stuff is built before window.insights is available

Hence this:
https://github.com/RedHatInsights/insights-chrome/blob/master/src/js/nav/globalNav.js#L164

Going to go ahead w/o that for now